proto/authzed/api/v1/00_handwritten_validation.go

```diff
@@ -9,6 +9,9 @@ func (m *CheckPermissionRequest) HandwrittenValidate() error {
 			reason: "alphanumeric value is required",
 		}
 	}
+	if m.GetSubject() != nil {
```

Comment:
I wonder if it'd be better to have this conditional inside the validation method so that callers can unconditionally call validate